app-modules/interaction/tests/Tenant/Filament/Actions/BulkCreateInteractionActionTest.php

```diff
@@ -0,0 +1,173 @@
+        ['interaction_relation_id' => 'required'],
+    ],
+    'interaction_relation_id exists' => [
+        BulkCreateInteractionActionRequestFactory::new()->state(['interaction_relation_id' => (string) Str::uuid()]),
+        ['interaction_relation_id' => 'exists'],
+    ],
+    'interaction_status_id required' => [
+        BulkCreateInteractionActionRequestFactory::new()->without('interaction_status_id'),
+        ['interaction_status_id' => 'required'],
+    ],
+    'interaction_status_id exists' => [
+        BulkCreateInteractionActionRequestFactory::new()->state(['interaction_status_id' => (string) Str::uuid()]),
+        ['interaction_status_id' => 'exists'],
+    ],
+    'interaction_type_id required' => [
+        BulkCreateInteractionActionRequestFactory::new()->without('interaction_type_id'),
+        ['interaction_type_id' => 'required'],
+    ],
+    'interaction_type_id exists' => [
+        BulkCreateInteractionActionRequestFactory::new()->state(['interaction_type_id' => (string) Str::uuid()]),
+        ['interaction_type_id' => 'exists'],
+    ],
+    'start_datetime required' => [
+        BulkCreateInteractionActionRequestFactory::new()->without('start_datetime'),
+        ['start_datetime' => 'required'],
+    ],
+    'end_datetime required' => [
+        BulkCreateInteractionActionRequestFactory::new()->without('end_datetime'),
+        ['end_datetime' => 'required'],
+    ],
+    'subject required' => [
+        BulkCreateInteractionActionRequestFactory::new()->without('subject'),
+        ['subject' => 'required'],
+    ],
+    'description required' => [
+        BulkCreateInteractionActionRequestFactory::new()->without('description'),
+        ['description' => 'required'],
+    ],
+]);
+
+it('can successfully create bulk interaction with student', function () {
+    asSuperAdmin();
+
+    $student = Student::factory()->create();
+
+    $request = BulkCreateInteractionActionRequestFactory::new()->create();
+
+    livewire(ListStudents::class)
+        ->mountTableBulkAction('createInteraction', [$student->getKey()])
+        ->setTableBulkActionData($request)
+        ->callMountedTableBulkAction()
+        ->assertHasNoTableBulkActionErrors()
+        ->assertSuccessful()
+        ->assertNotified();
+
+    assertDatabaseHas(Interaction::class, $student->interactions->first()->toArray());
+});
+
+it('can successfully create bulk interaction with prospect', function () {
+    asSuperAdmin();
+
+    $prospect = Prospect::factory()->create();
+
+    $request = BulkCreateInteractionActionRequestFactory::new()->create();
+
+    livewire(ListProspects::class)
+        ->mountTableBulkAction('createInteraction', [$prospect->getKey()])
+        ->setTableBulkActionData($request)
+        ->callMountedTableBulkAction()
+        ->assertHasNoTableBulkActionErrors()
+        ->assertSuccessful()
+        ->assertNotified();
+
+    assertDatabaseHas(Interaction::class, $prospect->interactions->first()->toArray());
```

Comment:
This does not provide any proper assertion.

It may be best to manually write out the attributes in an array here. Using the $request and the $student to confirm that the interaction was created as you would expect.

The way this is written now, it validates nothing other than that the database has a record that it has.